### DISCUSSION TITLE
Difficulty Binding Print Screen key to Flameshot in i3wm

### DISCUSSION THREAD

**Discussion by markpopham** [i3/i3#6038] [Q&A]
Hello!
I'm using arch with i3wm on a Lenovo Yoga laptop. I've got a little Print Screen key in the top corner of my keyboard and would love to bind it to flameshot; however, it doesn't seem like this laptop uses "Print" as the symcode for the key. 
Using 

`xev -event keyboard | egrep -o 'keycode.*\)'`

and hitting the key gives me not one keysym but several at once:

`keycode 133 (keysym 0xffeb, Super_L)
keycode 50 (keysym 0xffe1, Shift_L)
keycode 39 (keysym 0x53, S)
keycode 39 (keysym 0x53, S)
keycode 50 (keysym 0xffe1, Shift_L)
keycode 133 (keysym 0xffeb, Super_L)
`

In my i3 config file I've got

`bindsym` Super_L+Shift_L+S exec "flameshot full --path /home/user/Pictures/"

and have tried multiple configurations including using symcodes. Is this something that is unique to this particular laptop?

Thank you!

**Comment by pflingstring** [comment]
It looks to me like you are typing into `xev` the sequence of keys that you want to be the final shortcut. 
I'm not trying to be condescending, but are you *really* just pressing the single individual **PrtSc** key, without holding down anything else (like *shift* or *meta*)? Would you please try it again and see if it's the same output? Usually you'd get only 2 events, not 6...
I have remapped even the `power-up/shutdown` key, this one clearly was not intended to be used as a regular key for typing, but it works nonetheless.

Can you try this binding and see if it works?
`bindcode Mod4+107  exec "flameshot full --path /home/user/Pictures/"`

**Reply by markpopham** [reply]
Hahaha, no, I totally understand you're not trying to be condescending! It is very surprising - hitting JUST the PrtSc key gives the following:

`keycode 36 (keysym 0xff0d, Return)
keycode 133 (keysym 0xffeb, Super_L)
keycode 50 (keysym 0xffe1, Shift_L)
keycode 39 (keysym 0x53, S)
keycode 39 (keysym 0x53, S)
keycode 50 (keysym 0xffe1, Shift_L)
keycode 133 (keysym 0xffeb, Super_L)`

and each subsequent press gives

`keycode 133 (keysym 0xffeb, Super_L)
keycode 50 (keysym 0xffe1, Shift_L)
keycode 39 (keysym 0x53, S)
keycode 39 (keysym 0x53, S)
keycode 50 (keysym 0xffe1, Shift_L)
keycode 133 (keysym 0xffeb, Super_L)`

You can find a screencapture below (I bound it to another key for right now)

![2024-05-07_21-31](https://github.com/i3/i3/assets/81940058/31edfc27-36cb-4727-ac87-bb077b861bc4)

Unfortunately, the binding provided doesn't work.

**Comment by shamilbi** [comment]
from my config:
```
bindsym --release Print exec flameshot gui
```

**Comment by YAUIO** [comment]
from my config
`bindsym Print exec flameshot gui`
if that will help
